PocketKit/Sources/PocketKit/Home/HomeCarouselItemCell.swift

```diff
@@ -25,19 +25,22 @@ class HomeCarouselItemCell: UICollectionViewCell {
     private let titleLabel: UILabel = {
         let label = UILabel()
         label.numberOfLines = Constants.maxTitleLines
+        label.adjustsFontForContentSizeCategory = true
```

Comment:
Did this turn out to be strictly necessary? I would imagine that the font size is scaled when building the text attributes from the style here:
https://github.com/Pocket/pocket-ios/blob/d00f21443d7b4a6f97967d5b223743e06fa4ac8c/PocketKit/Sources/Textile/Style/Style%2BUIKit.swift#L141

Reply:
The fonts wouldn't scale if I background the app, changed the font size in Settings and then foreground the app. It does scale after closing and re-opening the app. Adding this change allows the fonts to adjust after the user navigates back to the app, without having to relaunch it.